src/python/pose_format/utils/generic.py

```diff
@@ -7,7 +7,36 @@
 from pose_format.pose_header import PoseHeader, PoseHeaderDimensions
 from pose_format.utils.normalization_3d import PoseNormalizer
 from pose_format.utils.openpose import OpenPose_Components
+from pose_format.utils.openpose_135 import OpenPose_Components as OpenPose135_Components
+# from pose_format.utils.holistic import holistic_components # creates an error: ImportError: Please install mediapipe with: pip install mediapipe
 
+SupportedPoseFormat = Literal["holistic", "openpose", "openpose_135"]
+
+def detect_pose_format(pose: Pose) -> SupportedPoseFormat:
```

Comment:
I would suggest `detect_known_pose_format` (because technically you can use the library for arbitrary poses)